lib/fog/libvirt/models/compute/server.rb

```diff
@@ -351,14 +351,18 @@ def defaults
           }
         end
 
+        def valid_boot_media
+          %w[cdrom fd hd network]
+        end
+
         def default_boot_order
           %w[hd cdrom network]
         end
 
         def verify_boot_order order = []
           if order
             order.each do |b|
-              raise "invalid boot order, possible values are: hd, network and/or cdrom" unless default_boot_order.include?(b)
```

Comment:
Well, if it is not used anywhere else i believe we can get rid of this method and push the value to the DSL definition of `boot_order` instead.

What do you think?